src/Microsoft.Azure.SignalR.AspNet/Middleware/NegotiateMiddleware.cs

```diff
@@ -209,7 +209,7 @@ private IEnumerable<Claim> BuildClaims(IOwinContext owinContext, IRequest reques
             var user = owinContext.Authentication?.User;
             var userId = _provider?.GetUserId(request);
 
-            var claims = ClaimsUtility.BuildJwtClaims(user, userId, GetClaimsProvider(owinContext), _serverName, _mode, _enableDetailedErrors, _endpointsCount);
+            var claims = ClaimsUtility.BuildJwtClaims(user, userId, GetClaimsProvider(owinContext), _serverName, _mode, _enableDetailedErrors, _endpointsCount, _disconnectTimeout);
```

Comment:
should we use builder pattern?
I find wangpeng also add an parameter in this build.

Reply:
Good idea. Will use another PR to do the refactor.